File: 557-reverse-words-in-a-string-iii/557-reverse-words-in-a-string-iii.cpp
class Solution {
public:
    string reverseWords(string s) 
    {
        string s2="", s3="", rev="";
        s+=' ';
        for(int i=0; i<s.length(); i++)
        {
            if(s[i]!=' ')
            {
                s2 = s[i] + s2;
            }
            else
            {
                if(i==s.length()-1)
                    s3 += s2;
                else 
                    s3 += s2 + " ";
                s2 = "";
            }
        }
        return s3;
    }
};

File: 557-reverse-words-in-a-string-iii/557-reverse-words-in-a-string-iii.java
class Solution 
{
    public String reverseWords(String s) 
    {
        String str[] = s.split(" ");
        String s2 = "";
        for(int i=0; i<str.length; i++)
        {
            StringBuilder input = new StringBuilder(str[i]);
            input.reverse();
            s2 += (i!=str.length-1 ? (input+" ") : input);            
        }
        return s2;
    }
}